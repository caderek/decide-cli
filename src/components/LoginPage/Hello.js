import React from 'react'
import $ from './Hello.scss'

const Hello = () => (
  <div className={$.hello}>
    <img src='logo.png' alt='Logo' />
    <h2>Prioritize your{'\u00A0'}ideas with your{'\u00A0'}team.</h2>
  </div>
)

export default Hello
