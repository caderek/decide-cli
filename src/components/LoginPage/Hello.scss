.hello {
  text-align: center;

  h2 {
    color: #70889C;
    margin-top: 15px;
    margin-bottom: 80px;
  }
}
