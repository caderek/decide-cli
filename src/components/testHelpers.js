import renderer from 'react-test-renderer'

export const snap = component => {
  const tree = renderer.create(component).toJSON()
  expect(tree).toMatchSnapshot()
}
