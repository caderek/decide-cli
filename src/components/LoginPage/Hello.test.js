import React from 'react'
import { snap } from '../testHelpers'
import Hello from './Hello'

test('renders correctly', () => snap(
  <Hello />
))
