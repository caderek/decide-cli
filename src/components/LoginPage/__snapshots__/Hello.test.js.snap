exports[`test renders correctly 1`] = `
<div
  className="hello">
  <img
    alt="Logo"
    src="logo.png" />
  <h2>
    Prioritize your
     
    ideas with your
     
    team.
  </h2>
</div>
`;
